### ISSUE TITLE
Weird behavior on Touch Mode

### ISSUE THREAD

**Issue by jjavierdguezas** [manojVivek/responsively-app#232]
>Version: 0.5.0
Electron: 8.3.0
Chrome: 80.0.3987.165
Node.js: 12.13.0
V8: 8.0.426.27-electron.0
OS: Windows_NT x64 10.0.18362

I was testing the new Touch Mode and I detected some issues, I'll try to explain here
- When Touch Mode is on, you cannot perform any action until you click in the area you want to do it (to drag in a device you must click in the device first, if you want to toggle touch mode again you need to click in the black area of the app)
- Sometimes the Inspect Element button activates itself
- When Touch Mode is on, the tooltip of it button looks bigger (when it appears)

![touchmodeissue](https://user-images.githubusercontent.com/13673443/86233643-02854b80-bb96-11ea-94d7-1c3bc939607f.gif)

**Comment by jjavierdguezas** [comment]
Oh I figured out that the Inspect Element button was not activating itself, the issue is more like this:
- When you click the Toggle Touch Mode button and Touch Mode is on, moving the mouse to the next button (left or right) will make it look hovered until you toggle Touch Mode again

![touchmodeissue2](https://user-images.githubusercontent.com/13673443/86234807-c521bd80-bb97-11ea-87ab-449752c6d93d.gif)

**Comment by manojVivek** [comment]
> Oh I figured out that the Inspect Element button was not activating itself, the issue is more like this:
> 
> * When you click the Toggle Touch Mode button and Touch Mode is on, moving the mouse to the next button (left or right) will make it look hovered until you toggle Touch Mode again
> 


@jjavierdguezas  Yeah, other icons getting activated is because when the mouse events are changed to touch events, the `unhovering` is not happening. Since hover is a mouse event. I couldn't think of a way to fix this.

**Comment by manojVivek** [comment]
> * When Touch Mode is on, you cannot perform any action until you click in the area you want to do it (to drag in a device you must click in the device first, if you want to toggle touch mode again you need to click in the black area of the app)

Yeah, I faced this too, for some reason the `webview` and the other parts of the app are losing focus, so that needs the extra tap? 

@romanakash Let us know if you have any ideas.

**Comment by romanakash** [comment]
Looks like another unresolved electron issue https://github.com/electron/electron/issues/19645

**Comment by manojVivek** [comment]
Oh! 😓 

Let's keep this open for tracking this issue.
